DQM/DTMonitorClient/python/dtTnPEfficiencyClient_cfi.py

```diff
@@ -0,0 +1,43 @@
+                                                                          "RPC_nPassingProbePerRoll_Endcap_Sta-3:RPC_nFailingProbePerRoll_Endcap_Sta-3",
+                                                                          "RPC_nPassingProbePerRoll_Endcap_Sta-2:RPC_nFailingProbePerRoll_Endcap_Sta-2",
+                                                                          "RPC_nPassingProbePerRoll_Endcap_Sta-1:RPC_nFailingProbePerRoll_Endcap_Sta-1",
+                                                                          "RPC_nPassingProbePerRoll_Endcap_Sta1:RPC_nFailingProbePerRoll_Endcap_Sta1",
+                                                                          "RPC_nPassingProbePerRoll_Endcap_Sta2:RPC_nFailingProbePerRoll_Endcap_Sta2",
+                                                                          "RPC_nPassingProbePerRoll_Endcap_Sta3:RPC_nFailingProbePerRoll_Endcap_Sta3",
+                                                                          "RPC_nPassingProbePerRoll_Endcap_Sta4:RPC_nFailingProbePerRoll_Endcap_Sta4",
+                                                                          "RPC_nPassingProbePerRoll_Endcap_1D_Sta-4:RPC_nFailingProbePerRoll_Endcap_1D_Sta-4",
+                                                                          "RPC_nPassingProbePerRoll_Endcap_1D_Sta-3:RPC_nFailingProbePerRoll_Endcap_1D_Sta-3",
+                                                                          "RPC_nPassingProbePerRoll_Endcap_1D_Sta-2:RPC_nFailingProbePerRoll_Endcap_1D_Sta-2",
+                                                                          "RPC_nPassingProbePerRoll_Endcap_1D_Sta-1:RPC_nFailingProbePerRoll_Endcap_1D_Sta-1",
+                                                                          "RPC_nPassingProbePerRoll_Endcap_1D_Sta1:RPC_nFailingProbePerRoll_Endcap_1D_Sta1",
+                                                                          "RPC_nPassingProbePerRoll_Endcap_1D_Sta2:RPC_nFailingProbePerRoll_Endcap_1D_Sta2",
+                                                                          "RPC_nPassingProbePerRoll_Endcap_1D_Sta3:RPC_nFailingProbePerRoll_Endcap_1D_Sta3",
+                                                                          "RPC_nPassingProbePerRoll_Endcap_1D_Sta4:RPC_nFailingProbePerRoll_Endcap_1D_Sta4",
+                                                                          "RPC_nPassingProbe_Endcap_allCh_1D:RPC_nFailingProbe_Endcap_allCh_1D",
+                                                                          "CSC_nPassingProbe_allCh:CSC_nFailingProbe_allCh",
+                                                                          "CSC_nPassingProbe_allCh_1D:CSC_nFailingProbe_allCh_1D",
+                                                                          "CSC_nPassingProbe_allCh:CSC_nFailingProbe_allCh",
```

Comment:
This entry is repetaed, is it intentional?

Reply:
> This entry is repeated, is it intentional?

@llunerti this question was never answered: could you please comment?

Reply:
Hi @perrotta , @jfernan2 I've somhow missed this comment, thanks for pingning me againg.

This file and DQM/DTMonitorModule/python/dtTnPEfficiencyTask_cfi.py are old configuration file, and they are now not being used. The new configuration file, that are the ones actually run, are now under DQMOffline/MuonDPG/python/.

I'm removing DQM/DTMonitorModule/python/dtTnPEfficiencyTask_cfi.py and DQM/DTMonitorClient/python/dtTnPEfficiencyClient_cfi.py in the next commit.